lib/adapters/local/find/in-memory-filter.js

```diff
@@ -78,6 +82,10 @@ function rowFilter (doc, selector, inMemoryFields) {
   });
 }
 
+function isDesignDoc (doc) {
+  return /_design/.test(doc._id);
```

Comment:
should be `/^_design\//` to start searching at the beginning of the string and include the `/`